scripts/zkai_attention_kv_stwo_native_two_head_bounded_softmax_table_proof_input.py

```
@@ -0,0 +1,787 @@
+                "weighted_value": list(weighted_value),
+                "weighted_numerator": list(numerators),
+                "attention_output": list(output),
+                "output_remainder": list(remainders),
+            })
+        current = candidates
+    if local_step_counts != [SEQUENCE_LENGTH for _ in range(HEAD_COUNT)]:
+        raise AttentionKvTwoHeadBoundedSoftmaxTableInputError("per-head input step count drift")
+    return rows, current, outputs
+
+
+def validate_payload(payload: dict[str, Any]) -> None:
```

Ruff (0.15.12)

[warning] Too many branches (19 > 12)

(PLR0912)